cli/command/manifest/fetch.go

```diff
@@ -44,23 +43,21 @@ func getLocalImageManifestData(namedRef reference.Named, transactionID string) (
 		return nil, nil, err
 	}
 
-	// If this is a manifest list, let's check for it locally so a user can see any modifications
-	// he/she has made.
+	// TODO: is this case relevant if transactionID == "" ?
```

Comment:
hah, ok. this is making me a little crazy. can you squash all these into another (temporary if you want) branch? :D i really want to get through all these by tomorrow!